From 281a78aade45ae933465015ef070ed6942ce03fa Mon Sep 17 00:00:00 2001
From: alexytomi <60690056+alexytomi@users.noreply.github.com>
Date: Mon, 22 Dec 2025 02:34:49 +0800
Subject: [PATCH 3/5] [Sideloader] Bump botan to master

Used `dub upgrade -s botan` after editing dub.json with the new value.

This works around some weird cross-compiling android-specific TLS bug
that I am not sure on the specifics of.
---
 frontends/gtk/dub.selections.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/frontends/gtk/dub.selections.json
+++ b/frontends/gtk/dub.selections.json
@@ -2,7 +2,7 @@
 	"fileVersion": 1,
 	"versions": {
 		"automem": "0.6.11",
-		"botan": "1.13.8",
+		"botan": "~master",
 		"botan-math": "1.0.4",
 		"cachetools": "0.4.2",
 		"dynamic-loader": {"version":"9202b389affa6be0740e7295dea8f0146537a784","repository":"git+https://github.com/Dadoum/dynamicloader.git"},
